.github/issue-close.yml

```diff
@@ -0,0 +1,17 @@
+# Config for Close Issue app: https://probot.github.io/apps/close-issue/
+
+# Comment that will be sent if an issue is judged to be closed
+comment: "This issue is closed because it does not meet any of our issue templates. Please make sure to use it."
```

Comment:
```suggestion
comment: "This issue was closed because it does not use any of our issue templates. Please make sure to use one of the suggested templates."
```